packages/rich-text-editor/src/RichTextEditor/controlmap.ts

```diff
@@ -32,18 +32,73 @@ type ControlGroupTypes = {
   [key in ToolbarControlTypes]?: string
 }
 
+/**
+ * Chains multiple commands to dispatch each transitions in sequential order
+ */
+function chainTransactions(...commands: Command[]): Command {
```

Comment:
Depending on whether this is helpful or not, we could potentially export this from the toolkit